README.md

```diff
@@ -1,2 +1,29 @@
-# index-mermaid-graph
-a tool that produces SVG diagrams using OLM graphs as the input
+## Output Mermaid graph script from Operator Lifecycle Manager indices
+
+### Required:
+make, sqlite3, go v14+, container execution environment (such as Docker, podman)
+
+### Usage
+Adjust the sqlite file pointed to in `sqlite3.sql`, this controls what index gets graphed
+
+Then:
+```bash
+make run <ARGS=operator-package-name>
+```
+For instance:
+```bash
+make run ARGS=amq-operator
+```
+or
+```bash
+INDEX_DB_PATH_AND_NAME=olm_catalog_indexes/index.db.4.7.redhat-operators make run ARGS=jaeger-product
+```
```

Comment:
When we should use the first option and/or the second one?

Reply:
cleaned up the wording